### DISCUSSION TITLE
Enable additional features only on one side

### DISCUSSION THREAD

**Discussion by adastx** [manna-harbour/miryoku#160] [Q&A] [answered]
I recently updated my firmware to take advantage of the tap layer for gaming:
![image](https://user-images.githubusercontent.com/56338480/199111353-7e68ad09-b975-47c8-987c-e262661e2a52.png)

This works great! 
But when typing, sometimes I might be too sloppy and accidentally activate one of these keys.

Would it be possible to enable the additional features only on the right side of the keyboard?
I think that would greatly lower the chances of accidental activation.

**Comment by manna-harbour** [comment, marked as answer]
See https://github.com/manna-harbour/miryoku/discussions/93.

In the mean time you could substitute custom layers with the Additional Features keys removed. See https://github.com/manna-harbour/miryoku/discussions/85.

**Reply by adastx** [reply]
Thanks for the quick reply!

Double tap does seems like a much better idea than what I was suggesting.

Will bootloader require double tap on QMK?

**Reply by manna-harbour** [reply]
For Miryoku QMK and Miryoku KMonad all Additional Features will require double tap. That's just a ZMK limitation.
